mod-orders-storage/schemas/order_template.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "order template",
+  "type": "object",
+  "properties": {
+    "name": {
```

Comment:
Please add `template` prefix to all the fields like mention in jira ticket. The reason is simple - just to make these props to not intersect in future with new properties of an order. For example, some of the library will request to add `code` property to an order and the template will not allow to specify both template's code and desired order's code...